axum-extra/CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Unreleased
 
-- None.
+- **fix:** Depend on tower with `default_features = false`
```

Comment:
PR reference missing again, or I'm reviewing too early ^^ (should this be automated?)